spec/helpers/markdown_helper_spec.rb

```diff
@@ -0,0 +1,10 @@
+require "rails_helper"
+
+RSpec.describe MarkdownHelper do
+  describe "#strip_markdown" do
```

Comment:
Could we add a test with content that have multiple links, ie:

`this is [a link](/blaaahhh) and this is also [a liiiiinnkkk](/bajajajja)`

Reply:
Sure!

Reply:
@DilwoarH Updated and fixed up.